src/Main.java

```diff
@@ -0,0 +1,166 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Main obj = new Main();
+
+
+
+        System.out.println("Метод findSymbolOccurance");
+
+        obj.findSymbolOccurance("Java", 'a');
+
+        System.out.println();
+
+        System.out.println("Метод findWordPosition");
+
+
+        obj.findWordPosition("Apollo", "pollo" );
+        obj.findWordPosition("Aplle", "Plant" );
+
+        System.out.println();
+
+        System.out.println("Метод stringReverse");
+        obj.stringReverse("Hello");
+
+        System.out.println();
+
+        System.out.println("Метод isPalindrome");
+        obj.isPalindrome("ERE");
+        obj.isPalindrome("Allo");
+
+        System.out.println();
+        System.out.println();
+
+        System.out.println("Гра - вгадай слово");
+        obj.guessWord();
+
+
+
+
+
+
+
+
+    }
+
+    public void findSymbolOccurance (String str, char ch) {
+
+        char[] charArray = str.toCharArray();
+        int count = 0;
+
+        for ( int i = 0; i < charArray.length; i++ ) {
+            if ( charArray[i] == ch ) {
+                count++;
+            }
+        }
+
+        System.out.println("Символ зустрічається : " + count + " раз/разів");
+    }
+
+
+    public void findWordPosition (String source, String target) {
```

Comment:
This method works incorrectly. You should return a position in a source string which is a start for target, for example, source = "Java is awesome", and target = "is", therefore the answer is 5. Why is that? Because 5 is the position of the letter 'i'. If the source string doesn't contain the target string, then return -1.